src/Nox.Solution/Models/Application/Localization/Localization.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using Nox.Types.Schema;
+
+namespace Nox.Solution;
+
+[GenerateJsonSchema]
+[Title("The localization settings for the solution.")]
+[Description("The supported localization and defaults for the solution.")]
+[AdditionalProperties(false)]
+public class Localization
+{
+    [Title("The list of culture codes supported by the solution.")]
+    [Description("The list of culture codes that the solution supports.")]
+    [Pattern(@"^[a-z]{2}-[A-Z]{2}$")]
+    [Required]
+    public IReadOnlyList<string> SupportedCultures { get; set; } = new List<string>() { "en-US" };
```

Comment:
could we use CultureCode type to get valitaion (5 letters)

Reply:
Good idea, but it would create a dependency on Nox.Types.